aten/src/ATen/native/CPUBlas.cpp

```diff
@@ -337,9 +338,20 @@ void gemm(
               b, &ldb_,
               &beta_,
               float_v.data(), &ldc_);
-      for (auto cv: float_v) {
-        *(c++) = c10::convert<at::BFloat16>(cv);
-      }
+      at::parallel_for(0, c_size, 1, [&](int64_t begin, int64_t end) {
```

Comment:
Usually the grain size would be `at::internal::GRAIN_SIZE` which avoids introducing threading overhead for very small tensors.